src/Approvals/SFA.DAS.Approvals/Application/Cohorts/Queries/GetCohortDetails/GetCohortDetailsQuery.cs

```diff
@@ -85,6 +87,7 @@ public async Task<GetCohortDetailsQueryResult> Handle(GetCohortDetailsQuery requ
             {
                 LegalEntityName = cohort.LegalEntityName,
                 ProviderName = cohort.ProviderName,
+                HasNoDeclaredStandards = providerCourses.Standards.IsNullOrEmpty(),
```

Comment:
I don't think we should be using this extension method here. Although it functions as we would hope it doesn't feel appropriate to pull in this `Microsoft.IdentityModel.Tokens` namespace. I'm surprised this extension is publicly accessible from that package.

Reply:
Done